commons-db/src/main/java/com/palantir/nexus/db/DBType.java

```diff
@@ -27,32 +27,9 @@
  * @author akashj
  */
 public enum DBType {
-    // AJ: the oracle jdbc url is missing a final "paren" - processing in DBMgr will fix this...
-    ORACLE("oracle.jdbc.driver.OracleDriver", "SELECT 1 FROM dual", true),
-    POSTGRESQL("org.postgresql.Driver", "SELECT 1", true),
-    H2_MEMORY("org.h2.Driver", "SELECT 1", true);
-
-    private final String driver;
-    private final String testQuery;
-    private final boolean hasGIS;
-
-    public boolean supportsGIS() {
-        return hasGIS;
-    }
-
-    DBType(String driver, String testQuery, boolean hasGIS) {
-        this.driver = driver;
-        this.testQuery = testQuery;
-        this.hasGIS = hasGIS;
-    }
-
-    public String getDriverName() {
-        return driver;
-    }
-
-    public String getTestQuery() {
```

Comment:
I think this is still used in large internal product?

Reply:
Ack, will revert this